packages/core/contracts/cross-chain-oracle/README.md

```diff
@@ -70,24 +71,26 @@ yarn hardhat --network arbitrum etherscan-verify --api-key <ETHERSCAN_KEY> --lic
 yarn hardhat --network boba sourcify
 # Optimism (etherscan-verify does not work on Optimism so we use hardhat verify)
 yarn hardhat --network optimism verify <EACH-DEPLOYED ADDRESS IN STEP 3> <ASSOCIATED CONSTRUCTOR PARAMS FROM 3>
+# Base
+yarn hardhat --network base verify <EACH-DEPLOYED ADDRESS IN STEP 3> <ASSOCIATED CONSTRUCTOR PARAMS FROM 3>
 ```
 
 5. Setup mainnet contracts
 
 ```sh
-yarn hardhat [setup-l1-arbitrum-xchain/setup-l1-boba-xchain/setup-l1-optimism-xchain] --network mainnet
+yarn hardhat [setup-l1-arbitrum-xchain/setup-l1-boba-xchain/setup-l1-optimism-xchain/setup-l1-base-xchain] --network mainnet
 ```
 
 6. Setup l2 contracts
 
 ```sh
-yarn hardhat setup-l2-xchain --network [arbitrum/boba/optimism]
+yarn hardhat setup-l2-xchain --network [arbitrum/boba/optimism/base]
 ```
 
-7. At this point, the cross chain contract suite setup is complete. We will now deploy the `OptimisticOracle` and required contracts to the L2.
+7. At this point, the cross chain contract suite setup is complete. We will now deploy the `OptimisticOracle,OptimisticOracleV2,OptimisticOracleV3` and required contracts to the L2.
```

Comment:
doesn't OptimisticOracleV3 depend on first having finder and whitelist setups as in later stages